target/linux/octeon/dts/cn7130_ubnt_edgerouter-4.dts

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "cn7130.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "ubnt,edgerouter-4", "cavium,cn7130";
+	model = "Ubiquiti EdgeRouter 4";
+
+	aliases {
+		/* White + Blinking Blue */
+		led-boot = &led_power_white;
+		/* Blue + Blinking White */
+		led-failsafe = &led_power_blue;
+		/* Constant Blue */
+		led-running = &led_power_blue;
+		/* Blue + Blinking White */
```

Comment:
Can you explain how these patterns are constructed?

Reply:
default settings is gpio15 (blue) high + gpio17 (white) high results in constant white + blue pattern.

> /* Constant Blue */
> led-running = &led_power_blue;

settings gpio15 (blue) low + gpio17 (white) high results in constant blue - that is how we get constant blue.

> /* White + Blinking Blue */
> led-boot = &led_power_white;

if we start triggering trigger white led we - get blinking blue light that looks like white led is there but blue is blinking.

> /* Blue + Blinking White */
> led-failsafe = &led_power_blue;
> /* Blue + Blinking White */
> led-upgrade = &led_power_blue;

if we start triggering trigger blue led we - get blinking white light that looks like blue led is there but white is blinking.

also there is this bonus (not used here):
if we set gpio15 (blue) low + gpio17 (white) low - it results in power light being completely off.